src/main/java/rocks/inspectit/gepard/agent/transformation/advice/InspectitAdvice.java

```diff
@@ -8,16 +8,17 @@
 public class InspectitAdvice {
 
   @Advice.OnMethodEnter(suppress = Throwable.class)
-  public static void onEnter(@Advice.AllArguments Object[] args, @Advice.This Object thiz) {
-    System.out.println("HELLO GEPARD");
+  public static void onEnter(@Advice.AllArguments Object[] args, @Advice.This Object thiz, @Advice.Origin String method) {
```

Comment:
The [OpenTelemetry Docs](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/writing-instrumentation-module.md#avoid-using-adviceorigin-method) don't recommend to use `@Advice.Origin` but instead to use loading constants from a constant pool.

So for the method, we can use `@Advice.Origin("#m") String methodName`